pkg/scheduler/scheduler.go

```diff
@@ -146,6 +146,15 @@ func calculateRequirementsForAssignments(log logr.Logger, workloads []workload.I
 			log.V(3).Info("Capacity not found when calculating workload assignments")
 			continue
 		}
+		ns := corev1.Namespace{}
+		if err := s.client.Get(ctx, types.NamespacedName{Name: w.Obj.Namespace}, &ns); err != nil {
```

Comment:
I'm curious when it would return an error here,  is the namespace not found? 
Ns will be deleted only after all object are deleted.

Reply:
may be connection errors as well, does the nature of the error matter here?